doc/COMPILING/COMPILING-MSYS.md

```diff
@@ -1,48 +1,50 @@
-# Compilation guide for 64 bit Windows (using MSYS2)
+# Compilation guide for 64-bit Windows (using MSYS2)
 
-This guide contains steps required to allow compilation of Cataclysm-DDA on Windows under MSYS2.
+This guide contains instructions for compiling Cataclysm-DDA on Windows under MSYS2. **PLEASE NOTE:** These instructions *are not intended* to produce a redistributable copy of CDDA. Please download the official builds from the website or [cross-compile from Linux](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/COMPILING/COMPILING.md#cross-compile-to-windows-from-linux) if that is your intention.
 
-Steps from current guide were tested on Windows 10 (64 bit) and MSYS2 (64 bit), but should work for other versions of Windows and also MSYS2 (32 bit) if you download 32 bit version of all files.
+These instructions were written using 64-bit Windows 7 and the 64-bit version of MSYS2; the steps should be the same for other versions of Windows.
 
 ## Prerequisites:
 
-* Computer with 64 bit version of modern Windows operating system installed (Windows 10, Windows 8.1 or Windows 7);
-* NTFS partition with ~10 Gb free space (~2 Gb for MSYS2 installation, ~3 Gb for repository and ~5 Gb for ccache);
-* 64 bit version of MSYS2 (installer can be downloaded from [MSYS2 homepage](http://www.msys2.org/));
+* Windows 7, 8, 8.1, or 10
+* NTFS partition with ~10 Gb free space (~2 Gb for MSYS2 installation, ~3 Gb for repository and ~5 Gb for ccache)
+* 64-bit version of MSYS2
 
 **Note:** Windows XP is unsupported!
 
 ## Installation:
 
-1. Go to [MSYS2 homepage](http://www.msys2.org/) and download 64 bit installer (e.g. [msys2-x86_64-20180531.exe](http://repo.msys2.org/distrib/x86_64/msys2-x86_64-20180531.exe)).
+1. Go to the [MSYS2 homepage](http://www.msys2.org/) and download the installer.
 
-2. Run downloaded file and install MSYS2 (click `Next` button, specify directory where MSYS2 64 bit will be installed (e.g. `C:\msys64`), click `Next` button again, specify Start Menu folder name and click `Install` button).
+2. Run the installer. It is suggested that you install to a dev-specific folder (C:\dev\msys64\ or similar), but it's not strictly necessary.
```

Comment:
Why it is suggested? MSYS 2 installer suggests a different path and I see no value in changing it to a longer path.

Reply:
I recommend this so that the user has the compiler and the code being compiled logically near each other on the drive, so that way they're not bouncing around their drive like a ping-pong ball. VCPKG's homepage even recommends this.

> First, download and bootstrap vcpkg itself; it can be installed anywhere, but generally we recommend using vcpkg as a submodule for CMake projects, and installing it globally for Visual Studio projects. We recommend somewhere like C:\src\vcpkg or C:\dev\vcpkg, since otherwise you may run into path issues for some port build systems.

Reply:
> We recommend somewhere like C:\src\vcpkg or C:\dev\vcpkg, since otherwise you may run into path issues for some port build systems.

I think it just means: use a short path without spaces.